config/tech-docs.yml

```diff
@@ -19,7 +20,7 @@ enable_search: true
 ga_tracking_id:
 
 # Enable collapsible navigation in the sidebar
-collapsible_nav: false
+collapsible_nav: true
```

Comment:
Possibly we also need `multipage_nav: true` here

Some context for the GDS template - originally it was developed as a single page, and the nav would be generated from that, but at some point there was functionality added to pick up a multi page structure

Based on the existence of subdirectories, I'm assuming that's what we want, but I haven't got it working locally yet

There is another example here that might be useful https://github.com/alphagov/tech-docs-gem/blob/main/example/config/tech-docs.yml

Reply:
Great suggestion. I've added this, and it _looks_ to be working locally